build/packer/ansible/agent-aws.yml

```diff
@@ -276,15 +301,15 @@
         name: fs.inotify.max_user_watches
         value: 1048576
 
-    - name: Get disk UUID
-      command: blkid -s UUID -o value /dev/nvme0n1p1
-      register: disk_uuid
+          #- name: Get disk UUID
```

Comment:
I'd remove these comments to keep things clean. We can get them from the history if necessary.